e2e-tests/cypress/tests/integration/channels/accessibility/accessibility_sidebar_spec.ts

```diff
@@ -11,6 +11,7 @@
 // Group: @channels @accessibility @smoke
 
 import * as TIMEOUTS from '../../../fixtures/timeouts';
+import 'cypress-real-events';
```

Comment:
Please move to https://github.com/mattermost/mattermost/blob/master/e2e-tests/cypress/tests/support/index.js to make it globally available.

Reply:
And probably should add `cypress-real-events` to `tsconfig.json`.

Reply:
done!